ringbuffer.hpp

```diff
@@ -63,6 +63,17 @@ namespace jnk0le
 			}
 
 			/*!
+             * \brief For debug purposes. Outputs data_buff array in full
+             */
+            void print()
```

Comment:
1. This will produce a compiler error for files that include this header that do not include <iostream> for including this header
2. Since this library is OS independent, embedded applications could use this, and <string> is notoriously large and not typically used, which is included by <iostream>